src/analysis/retail/shaman/enhancement/modules/Abilities.tsx

```diff
@@ -413,7 +413,7 @@ class Abilities extends CoreAbilities {
       },
       {
         // This is no error. We actually use the elemental shaman elemental blast spell id.
-        spell: TALENTS_SHAMAN.ELEMENTAL_BLAST_ENHANCEMENT_TALENT.id,
+        spell: TALENTS_SHAMAN.ELEMENTAL_BLAST_ELEMENTAL_TALENT.id,
         enabled: combatant.hasTalent(TALENTS_SHAMAN.ELEMENTAL_BLAST_ENHANCEMENT_TALENT),
         charges:
           combatant.getRepeatedTalentCount(TALENTS_SHAMAN.ELEMENTAL_BLAST_ENHANCEMENT_TALENT) +
```

Comment:
This line should also use the elemental version

Reply:
Why? The talent id is correct since it's used in hasTalent - so why would it not be accurate in getRepeatedTalentCount?

Reply:
getRepeatedTalentCount behaviour has been patched, should be accurate now.